Cargo.toml

```diff
@@ -12,11 +12,17 @@ crate-type = ["cdylib"]
 [dependencies]
 log = "0.4"
 itertools = "0.8"
-ironoxide = {version = "0.18.0", features = ["blocking"]}
+ironoxide = {version = "0.19.0", features = ["blocking"]}
 chrono = "0.4"
 serde_json = "~1.0"
 
 [build-dependencies]
 env_logger = "0.7"
 rust_swig = { git = "https://github.com/Dushistov/rust_swig.git", rev = "cfd12fb7406f9f2774832c051ce7b4e6c2e193d7"}
 bindgen = "0.52"
+
+[profile.dev]
+opt-level = 0
+
+[profile.dev.package."*"]
+opt-level = 3
```

Comment:
This speeds up running the scala tests significantly while not slowing down development time (unless you use `cargo clean` a lot).

Reply:
Adding this brings running the tests on my machine from about 160s to about 75s

Reply:
I had to remove this as it doubled our Travis build time. :-1: